views/email/email_partials/breach_alert.hbs

```diff
@@ -1,6 +1,6 @@
     <tr>
       <td valign="top" style="padding-top: 3px; padding-bottom: 10px; width: 70px;">
-        <img src="{{ SERVER_URL }}/img/logos/{{ breachAlert.Name }}.png" width="50" style="width: 50px;" />
+        <img src="{{ SERVER_URL }}/img/email_images/missing-logo-icon.png" width="50" style="width: 50px;" />
```

Comment:
Crap, I hit the wrong button to merge this when I think I want to request a change.

Reply:
Oh, no I mis-read where the image was being used. This is fine.